lms/djangoapps/courseware/tests/test_password_reset.py

```diff
@@ -0,0 +1,117 @@
+"""
+This file will test through the LMS some of the password reset features
+"""
+from uuid import uuid4
+
+import ddt
+from django.contrib.auth.models import User
+from django.contrib.auth.tokens import default_token_generator
+from django.test.utils import override_settings
+from django.utils.http import int_to_base36
+
+from courseware.tests.helpers import LoginEnrollmentTestCase
+from util.password_policy_validators import create_validator_config
+
+
+@ddt.ddt
+class TestPasswordReset(LoginEnrollmentTestCase):
```

Comment:
Is this a new Test class or was it moved from somewhere?

Reply:
It was moved/renamed from a TestPasswordHistory class that held some reset tests that had nothing to do with PasswordHistory.